MainModule/Server/Shared/Changelog.lua

```diff
@@ -3,18 +3,25 @@ return {
 	"*Drag edges to expand*";
 	"*Report bugs/issues on our GitHub repository*";
 	"";
-	"[v241 2023-12-18 16:00 UTC] @Dimenpsyonal";
-	"Command Box now scrolls (#1307)";
-	":BuyItem now only blocks limiteds (#1223)";
-	"Coasterteam various fixes and improvements (#1242)";
-	"Remove dropper from loader (#1238, #1280)";
-	":clone and :loadavatar are now the same command (#1198)";
-	"Add TextChatService support for command and command autocompletion (#1208)";
-	"Add :sing command (#1217)";
-	"Add new BanMenu UI; add functions+data models to the window code (#1295)";
-	":vote command is now in random order (#1179)";
-	"Add :orderedvote command for voting in order (#1179)";
-	"";
+	"[v241 2023-12-20 23:59 UTC] Portions (@Dimenpsyonal & @ccuser44 & @moo1210)";
+	"The Fixed & Bricked Update";
+	"Selene, rojo, aftman and git fixes & upgrades";
+	"";
+	"(Git/kaiserandaxl) Command Box now scrolls (#1307)";
+	"(Git/ccuser44) :BuyItem now only blocks limiteds (#1223)";
+	"(Git/coasterteam) Coasterteam various fixes and improvements (#1242)";
+	"(Git/chesse20) Remove dropper from loader (#1238)";
+	"(Git/ccuser44) Remove dropper artifacts, fix Adonis being totally broken (#1280)";
+	"(Git/ccuser44) :clone and :loadavatar are now the same command (#1198)";
+	"(Git/ccuser44) Add TextChatService support for command and command autocompletion (#1208)";
+	"(Git/ccuser44) Add :sing command (#1217)";
+	"(Git/kaiserandaxl) Add new BanMenu UI; add functions+data models to the window code (#1295)";
+	"(Git/karl-police) :vote command is now in random order (#1179)";
+	"(Git/ccuser44) Add :orderedvote command for voting in order (#1179)";
+	"(Git/dylanjkl) Implemented way to monitor Adonis Teleport status via a Character Attribute named \"ADONIS_isTeleporting\" (#1312)";
+	"(Git/kaiserandaxl) Added all properties and classes to \":GuiView\" (#1314)";
+	"(Git/WalkerOfBacon) Fix :sceleratis (#1305)";
+	"(Git/WalkerOfBacon) Fix :removestats (#1309)";
 	"(Git/ccuser44) Add command to debug anti cheat (#1267)";
 	"(Git/ccuser44) Fix remotespy detection (#1218)";
 	"(Git/ccuser44) Disable protections superseded by RejectCharacterDeletions on new games (#1245)";
```

Comment:
Why are changelogs always `git log` that's not a changelog

Reply:
What's the issue here?

Reply:
At least half of the commit messages make sense this time I think that might be a new record

Reply:
You are free to suggest changes